experimental/terraform/cloudbuild-webhook-notification/main.tf

```diff
@@ -0,0 +1,48 @@
+# notification configuration file generation
+data "template_file" "http_notification" {
+  template = file("${path.module}/templates/http.tpl")
+  vars = {
+    filter = "build.status == Build.Status.SUCCESS && build.substitutions[\"BRANCH_NAME\"] == \"${var.branch}\" && build.build_trigger_id == \"${var.trigger_id}\""
+    url    = var.url
+  }
+}
+# upload notification configuration file to storage bucket
+resource "google_storage_bucket_object" "http_notification" {
+  name    = "configuration/http-notification-for-${var.trigger_name}.yaml"
+  content = data.template_file.http_notification.rendered
+  bucket  = var.storage_bucket.name
+}
+# deploy cloud run notification service
+resource "google_cloud_run_service" "http_notification" {
+  name     = "http-notifier-for-${var.trigger_name}"
+  location = var.google_region
+
+  template {
+    spec {
+      containers {
+        image = "us-east1-docker.pkg.dev/gcb-release/cloud-build-notifiers/http:latest"
```

Comment:
does it make sense to use our own image instead?

Reply:
Hey @marcoths, I don't see any advantage to having our own image as we are just configuring the cloud build notifier. Also, this would increase complexity without adding any value at this point in time as I have just followed the documentation.

Reply:
okay, I guess we wouldn't have any problems with rate limiting then?

Reply:
These Cloud Run containers are internal to the project, so I don't see a need to rate limit.